runhouse/servers/cluster_servlet.py

```diff
@@ -71,6 +72,12 @@ async def aget_cluster_config(self) -> Dict[str, Any]:
         return self.cluster_config
 
     async def aset_cluster_config(self, cluster_config: Dict[str, Any]):
+        if "has_cuda" not in cluster_config.keys():
+            cpu_or_cuda = detect_cuda_version_or_cpu()
+            cluster_config["has_cuda"] = (
```

Comment:
shouldn't this be a boolean?

Reply:
Yeah, makes more sense that it will be a boolean.  
@mkandler In case you need to know on the frontend side whether the cluster has GPU, you can check if `cluster_config["has_cuda"] == True`